lib/views/log-message.js

```diff
@@ -5,42 +5,54 @@ import etch from 'etch'
 import path from 'path'
 import { Range } from 'atom'
 
+function fileRef (filePath, range, bracket) {
```

Comment:
I am going to try to simplify the function by using CSS classes for the brackets.